### DISCUSSION TITLE
LastModifiedTime is not working as expected when we define it inside the Filter

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2854] [Q&A]
We have added the LastModifiedTime inside our filters >> but it is not working, for example i defined to get the files which was modified from 13 March till 15 March >> i got zero results:-

![image](https://user-images.githubusercontent.com/79726511/225257285-04ce334a-8336-4836-be48-278b52e2c678.png)

Although we have more than 15 files added yesterday.. any advice?

**Comment by wobba** [comment]
1. Try to repro in OOB search UX - to check that files are searchable
2. Look at the request JSON being sent to see if the query makes sense or not
3. Replicate the query in e.g. SharePoint Query Tool or Graph Explorer depending on the provider you use
